resources/views/layouts/sidebars/_quick-links.blade.php

```diff
@@ -0,0 +1,34 @@
+@foreach ($currentCampaign->menuLinks()->with(['target'])->ordered()->get() as $menuLink)
+    <?php /** @var \App\Models\MenuLink $menuLink */ ?>
+    @if (isset($childName) && $menuLink->parent == $childName || !isset($childName) && $menuLink->parent == $name || $menuLink->parent == $name && $menuLink->parent == $childName )
+        @if ($menuLink->dashboard && $currentCampaign->boosted() && $menuLink->isValidDashboard())
```

Comment:
The rendering is copy-pasted across two blade files, would be nicer to have them in a single space